src/main/java/de/zalando/tip/zalenium/proxy/BrowserStackRemoteProxy.java

```diff
@@ -117,17 +119,19 @@ public String getCloudTestingServiceUrl() {
     @Override
     public TestInformation getTestInformation(String seleniumSessionId) {
         // https://BS_USER:BS_KEY@www.browserstack.com/automate/sessions/SELENIUM_SESSION_ID.json
-        String browserStackTestUrl = "https://%s:%s@www.browserstack.com/automate/sessions/%s.json";
-        browserStackTestUrl = String.format(browserStackTestUrl, getUserNameValue(), getAccessKeyValue(), seleniumSessionId);
+        String browserStackBaseTestUrl = "https://%s:%s@www.browserstack.com/automate/sessions/";
+        browserStackBaseTestUrl = String.format(browserStackBaseTestUrl, getUserNameValue(), getAccessKeyValue());
+        String browserStackTestUrl = browserStackBaseTestUrl + String.format("%s.json", seleniumSessionId);
         JsonObject testData = getCommonProxyUtilities().readJSONFromUrl(browserStackTestUrl).getAsJsonObject();
         String testName = testData.getAsJsonObject("automation_session").get("name").getAsString();
```

Comment:
Please extract `testData.getAsJsonObject("automation_session")` because it appears too many times 😄